src/WalletService/WalletServiceConfiguration.h

```
@@ -285,6 +285,86 @@ namespace PaymentService {
     }
   };
 
+   inline void handleIniConfig(std::ifstream& data, WalletServiceConfiguration& config){
+    // find key=value pair, respect whitespace before/after "="
+    // g0: full match, g1: match key, g2: match value
+    static const std::regex cfgItem{R"x(\s*(\S[^ \t=]*)\s*=\s*((\s?\S+)+)\s*$)x"};
+    
+    // comments, first non space starts with # or ;
+    static const std::regex cfgComment{R"x(\s*[;#])x"};
+    std::smatch item;
+    std::string cfgKey;
+    std::string cfgValue;
+
+    for (std::string line; std::getline(data, line);)
+    {
+      if (line.empty() || std::regex_match(line, item, cfgComment))
+      {
+        continue;
+      }
+      else if (std::regex_match(line, item, cfgItem))
+      {
+        if (item.size() == 4) {
+          cfgKey = item[1].str();
+          cfgValue = item[2].str();
+          
+          if(cfgKey.compare("daemon-address") == 0)
+          {
+            config.daemonAddress = cfgValue;
+          }
+          else if (cfgKey.compare("daemon-port")  == 0)
+          {
+            config.daemonPort = std::stoi(cfgValue);
```

std::stoi throws std::invalid_argument if given a non numeric argument - I guess this gets caught in ConfigurationManager, but we should probably catch it here so we can report a better error. It would be good to report the line it is failing to parse on, for easier debugging.